requirements.txt

```diff
@@ -1,15 +1,9 @@
-shortuuid==0.4.3
 pystache==0.5.4
-configobj==5.0.6
 wikipedia==1.4.0
 requests==2.13.0
-pyOpenSSL==16.2.0
```

Comment:
pyOpenSSL is used, I think websockets depend on it (and/or requests). For certain OS's the 16.0.0 package will cause problems and 16.2.0 or 17.0.0 is needed to make mycroft work across platforms.

We can probably remove it and let it be installed implicitly when the package requiring it is installed but it might cause problems if an older OS is upgraded.